packages/orbit-components/src/Modal/Modal.stories.tsx

```diff
@@ -89,6 +89,23 @@ Sizes.story = {
   },
 };
 
+export const ModalFooterOnly = () => {
+  const isBackButton = boolean("isBackButton", false);
+  const isNextButton = boolean("isNextButton", true);
```

Comment:
I understand this story in the context of this problem and this PR. But having these knobs makes it seem they're props from the component. Moreover, if anything, they should be called `hasBackButton`.

I do agree to have some kind of test to the problem fixed by this PR, but in the form of a test, not of a story.